core/src/Revolution/modElement.php

```diff
@@ -1157,6 +1157,36 @@ public function isStaticSourceValidPath()
         return $isValid;
     }
 
+    /**
+     * Get the absolute path to the preview file for this instance.
+     *
+     * @return string
+     */
+    public function getPreviewUrl() {
+        if (!empty($this->get('preview_file'))) {
+            $previewfile = $this->get('preview_file');
+
+            if ($this->get('source') > 0) {
```

Comment:
Turns out a media source needs to be assigned to the template for the preview thumb to be displayed.